tests/robot-tests/tests/admin_and_public/bau/publish_methodology.robot

```diff
@@ -28,7 +28,7 @@ Approve a methodology for publishing immediately
     user adds text block to editable accordion section    Methodology content section 1
     ...    ${METHODOLOGY_CONTENT_EDITABLE_ACCORDION}
     user adds content to accordion section text block    Methodology content section 1    1
-    ...    Adding Methodology content    ${METHODOLOGY_CONTENT_EDITABLE_ACCORDION}
+    ...    Adding Methodology content    ${METHODOLOGY_CONTENT_EDITABLE_ACCORDION}    Save
```

Comment:
Minor quibble, but why this keyword is taking `Save` as an argument isn't immediately clear if someone was reading this test suite alone. To save a user clicking through to understand what this argument is doing when they don't really need to, I think it would make the tests a little more readable if we instead used a new keyword like `user adds methodology content accordion text block` (and an associated keyword for releases).